backend/api/v1/endpoints/chat.py

```
@@ -190,37 +209,25 @@ async def upload_file(
     try:
         # Validate file type
         allowed_extensions = {".jpg", ".jpeg", ".png", ".gif", ".pdf", ".docx"}
-        # Get original extension but don't trust it for the actual path construction
+        # Get extension but only use it for validation
         original_ext = os.path.splitext(file.filename)[1].lower()
         if original_ext not in allowed_extensions:
             raise HTTPException(status_code=400, detail="Unsupported file type")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)